developers/weaviate/starter-guides/compression.mdx

```diff
@@ -59,35 +59,35 @@ These compression algorithms all help to control costs the same way. They reduce
 
 Compressed indexes use less RAM when they are loaded into memory, however they also use more disk space than uncompressed vectors. Weaviate stores the uncompressed vector and the compressed vector index. This means increased disk storage costs. However, since the cost of RAM is orders of magnitude higher than the cost of disk, the overall cost to use a compressed index is much lower than the cost of using an uncompressed index.
 
-The cost savings are obvious with in-memory indexes such as HNSW. The greater the RAM reduction, the smaller the costs.
+The cost savings are obvious with in-memory indexes such as HNSW. The greater the RAM reduction, the lower the requirements and thus the cost.
```

Comment:
I think it would be useful to highlight where the "less RAM" and thus reductions in cost occur. I think it's helpful to highlight that the reduction is in system **requirements**, and therefore not needing to provision as much resources.